lib/game_start.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module GameStart
```

Comment:
why do need this module? It's login should be in class Game

Reply:
It doesn't have login in the module. i don't understand your question.

Reply:
the methods in this module should be placed in Game class